bin/node/cli/tests/build_spec_works.rs

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020 Parity Technologies (UK) Ltd.
+// This file is part of Substrate.
+
+// Substrate is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Substrate is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
+
+#![cfg(unix)]
```

Comment:
Why would this not run on windows? I would expect the test to also run on windows, if not we should probably file an issue for it and reference it here as a FIXME...

If not, I suggest removing this line

```suggestion
```

Reply:
Testing on Windows was complicated but you're right that this test should work on Windows. I will create a ticket for that as I don't have time to test.

Reply:
Created #5018